vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/registry/impl/AbstractPackageRegistry.java

```diff
@@ -59,6 +65,17 @@
      */
     public static final String DEFAULT_PACKAGE_ROOT_PATH_PREFIX = DEFAULT_PACKAGE_ROOT_PATH + "/";
 
+    protected final String[] additionalAuthorizableIdsAllowedToExecuteHooks;
+
+    protected final String[] additionalAuthorizableIdsAllowedToInstallPackagesRequiringRoot;
```

Comment:
I think those a very long field names :-)

Reply:
Indeed, but I like verbosity over short names. Do you have any other proposal?